test/functional/feature_block.py

```diff
@@ -82,10 +82,7 @@ def set_test_params(self):
     def run_test(self):
         node = self.nodes[0]  # convenience reference to the node
 
-        # reconnect_p2p() expects the network thread to be running
-        network_thread_start()
-
-        self.reconnect_p2p()
+        self.bootstrap_p2p()  # Add one p2p connection to the node
```

Comment:
Note to reviewers: The updates in feature_block.py just tweak the network_thread_start/network_thread_join calls to happen in a more logical order, and be consistent with other tests. This is internal test cleanup which is not directly related to the other changes in this PR.